Source/SwiftLintFramework/Rules/PrefixedTopLevelConstantRule.swift

```diff
@@ -54,6 +54,11 @@ public struct PrefixedTopLevelConstantRule: ASTRule, OptInRule, ConfigurationPro
     public func validate(file: File,
                          kind: SwiftDeclarationKind,
                          dictionary: [String: SourceKitRepresentable]) -> [StyleViolation] {
+        if self.configuration.onlyPrivateMembers,
```

Comment:
Maybe this could be combined with the other `guard` below, since both are used for early exiting.

Reply:
I thought about that but since we only want to take into account if the ACL is private if this configuration is set, it felt like it was getting too complicated to be in that one as well.